catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -461,17 +464,17 @@ public void run() {
                       get_request_description());
                 }
               } catch (InvalidInputParameter invalidInputParameter) {
-                LOGGER.debug("Unable to notify callback", invalidInputParameter);
+                LOGGER.debug(UNABLE_TO_NOTIFY_CALLBACK, invalidInputParameter);
               } catch (ProcessingFault processingFault) {
-                LOGGER.debug("Unable to notify callback", processingFault);
+                LOGGER.debug(UNABLE_TO_NOTIFY_CALLBACK, processingFault);
               } catch (SystemFault systemFault) {
-                LOGGER.debug("Unable to notify callback", systemFault);
+                LOGGER.debug(UNABLE_TO_NOTIFY_CALLBACK, systemFault);
```

Comment:
✏️  These could be collapsed into (We can always do in a later PR)
```java
...
} catch (InvalidInputParamter | ProcessingFault | SystemFault failureException) {
  LOGGER.debug(UNABLE_TO_NOTIFY_CALLBACK, failureException);
}
```

Reply:
Collapsed 👍